Client/src/Components/TabPanels/ProfileSettings/ProfilePanel.jsx

```diff
@@ -0,0 +1,294 @@
+              <Button
+                level="error"
+                label="Delete account"
+                onClick={() => setIsOpen(true)}
+                sx={{
+                  fontSize: "13px",
+                  "&:focus": {
+                    outline: "none",
+                  },
+                }}
+              />
+            </Box>
+          </Stack>
+        </div>
+      </form>
+      <Divider
+        aria-hidden="true"
+        width="0"
+        sx={{ marginY: theme.spacing(6.25) }}
+      />
+      <Stack direction="row" justifyContent="flex-end">
+        <Box width="fit-content">
+          <ButtonSpinner
+            level="primary"
+            label="Save"
+            onClick={handleSaveProfile}
+            isLoading={isLoading}
+            loadingText="Saving..."
+            sx={{
+              paddingX: "40px",
+              height: "fit-content",
+              fontSize: "13px",
+              "&:focus": {
+                outline: "none",
+              },
+            }}
+          />
+        </Box>
+      </Stack>
+      {/* TODO - Update ModalPopup Component with @mui for reusability */}
+      <Modal
+        aria-labelledby="modal-delete-account"
+        aria-describedby="delete-account-confirmation"
+        open={isOpen}
+        onClose={() => setIsOpen(false)}
+        disablePortal
+      >
+        <Stack
+          gap="10px"
+          sx={{
+            position: "absolute",
+            top: "50%",
+            left: "50%",
+            transform: "translate(-50%, -50%)",
+            width: 400,
+            bgcolor: "white",
+            border: "solid 1px #f2f2f2",
+            borderRadius: "4px",
+            boxShadow: 24,
+            p: "30px",
+            "&:focus": {
+              outline: "none",
+            },
+          }}
+        >
+          <Typography id="modal-delete-account" variant="h4" component="h1">
+            Really delete this account?
+          </Typography>
+          <Typography
+            id="delete-account-confirmation"
+            variant="h5"
+            component="p"
+            sx={{
+              color: theme.palette.secondary.main,
+              fontSize: "13px",
+            }}
+          >
+            If you delete your account, you will no longer be able to sign in,
+            and all of your data will be deleted. Deleting your account is
+            permanent and non-recoverable action.
+          </Typography>
+          <Stack direction="row" gap="10px" mt="10px" justifyContent="flex-end">
+            <Button
+              level="tertiary"
+              label="Cancel"
+              onClick={() => setIsOpen(false)}
+              sx={{ fontSize: "13px" }}
+            />
+            <ButtonSpinner
+              level="error"
+              label="Delete account"
+              onClick={handleDeleteAccount}
+              isLoading={isLoading}
+              sx={{ fontSize: "13px" }}
```

Comment:
There are quite a few hardcoded styles and values throughout the components/pages.

For maintainability and flexiblity we should try to define these ideally in the `Theme` if they are shared across pages, or in the speciic page's CSS file if it is sepcific to the component/page only.

Reply:
Yeah I can do that. Wasn't quite sure if we were primarily gonna use `Theme` for styling or not so I was torn between using the two.

Reply:
I think rule of thumb is if it's being used across multiple components add it to the theme, otherwise if it's component specific it goes in the components css file